src/main/java/seedu/address/model/group/GroupName.java

```diff
@@ -57,7 +57,7 @@ public boolean equals(Object other) {
         }
 
         seedu.address.model.group.GroupName otherName = (seedu.address.model.group.GroupName) other;
-        return groupName.equals(otherName.groupName);
+        return groupName.equalsIgnoreCase(otherName.groupName);
```

Comment:
Good naming convention here! Good to include the ignore case to illustrate this point